molo/core/models.py

```diff
@@ -1493,12 +1493,17 @@ def get_effective_image(self):
             return page.specific.get_effective_image()
 
     def get_parent_section(self, locale=None):
-        page = SectionPage.objects.all().ancestor_of(self).last()
```

Comment:
Why does this not work anymore?

Reply:
It only gets the main language section, whereas the article itself is in a different language, this is the error we are seeing on bangladesh where the parent section page is in english and we are trying to index a bangladeshi page

Reply:
@codiebeulaine I just mean this one line, not the whole method. Why are you now getting the direct parent page rather than the last ancestor of the page. Was the last ancestor method being inconsistent or broken in some way?

Reply:
I'm just wondering what's wrong with using `ancestor_of()`?